consoleme/lib/s3_helpers.py

```diff
@@ -12,19 +12,27 @@
 from cloudaux.aws.sts import boto3_cached_conn
 
 from consoleme.config import config
+from consoleme.exceptions.exceptions import MissingConfigurationValue
 from consoleme.lib.plugins import get_plugin_by_name
 
 log = config.get_logger("consoleme")
 stats = get_plugin_by_name(config.get("plugins.metrics", "default_metrics"))()
 
 
 async def is_object_older_than_seconds(
-    bucket: str, key: str, older_than_seconds: int, s3_client=None
+    bucket: Optional[str], key: str, older_than_seconds: int, s3_client=None
```

Comment:
Should this function signature changed to make `bucket` a kwarg? The call is kind of awkward with an optional positional:

```python
is_object_older_than_seconds(None, 'a/key.json', 1234)
```

On the other hand, the kwarg approach is a tad awkward when you _do_ specify a bucket:

```python
is_object_older_than_seconds('this/is/the/key.json', 1234, bucket='a-bucket')
```